### PR TITLE
Feature/#54 add app user endpoint with jwt

### PR DESCRIPTION
**Pull request by dotnetdew**
User endpoint and Jwt Authentication

### CODE REVIEW

File: src/Infrastructure/Context/ApplicationDbContext.cs [hunk 1]
```diff
@@ -20,5 +20,25 @@ public ApplicationDbContext(DbContextOptions<ApplicationDbContext> options)
         public DbSet<ParkingPlace> ParkingPlaces { get; set; }
         public DbSet<Vacation> Vacations { get; set; }
         public DbSet<WorkPlace> WorkPlaces { get; set; }
+
+        public async override Task<int> SaveChangesAsync(CancellationToken cancellationToken = new CancellationToken())
```
Comment (svadrif) [inline src/Infrastructure/Context/ApplicationDbContext.cs:24]: put override before async keyword

File: src/API/Properties/launchSettings.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
       "dotnetRunMessages": true,
       "launchBrowser": true,
       "launchUrl": "swagger",
-      "applicationUrl": "http://localhost:8443;https://localhost:443",
+      "applicationUrl": "http://localhost:8443",
```
Comment (svadrif) [inline src/API/Properties/launchSettings.json:17]: return https://localhost:443 back

**Comment by dotnetdew** [comment]
closed

**Comment by dotnetdew** [comment]
closed

**Pull request closed without merging**